wcc-contentful-app/lib/wcc/contentful/model/section_contact_form.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class WCC::Contentful::Model::SectionContactForm < WCC::Contentful::Model
+  def send_email(data)
+    ::WCC::Contentful::App::UserMailer.contact_form_email(notificationEmail, data).deliver
+
+    save_contact_form(data)
+  end
+
+  def page
+    ::WCC::Contentful::Model::Page.find_by(sections: { id: id })
+  end
+
+  private
+
+  def save_contact_form(data)
+    return unless ::WCC::Contentful::App.db_connected?
+    return unless ::ActiveRecord::Base.connection
+      .table_exists? 'wcc_contentful_app_contact_form_submissions'
+
+    ::WCC::Contentful::App::ContactFormSubmission.create!(
+      full_name: data['First and Last Name'],
```

Comment:
I think the fact that these are hardcoded leans towards changing the implementation of the contact form fields we use here. Sorry - this is late, but Gordon and I were talking about this and remembered a conversation @frizman21 said we should go with the more simple  (less configurable) approach.

See https://github.com/watermarkchurch/theporch-live/blob/develop/db/migrate/20180905202327_create_simple_mailchimp_form.ts and other relevant files... shouldn't be too much work to modify

Reply:
Let's have a conversation before we re-open this...